test-data/unit/fine-grained-modules.test

```diff
@@ -971,3 +971,56 @@ x = Foo()
 ==
 ==
 main:2: error: Too few arguments for "foo" of "Foo"
+
+[case testAddModuleAfterCache1]
```

Comment:
Would it make sense to use the `-skip-nocache` suffix for some of these test cases?

Reply:
I don't see any need to. The main thing under test is cache behavior but they are still tests of module adding, which is subtle and could potentially find regressions in non-cache mode.